### DISCUSSION TITLE
Update-DbaInstance with security patch

### DISCUSSION THREAD

**Discussion by KezHalls** [dataplat/dbatools#7092] [Q&A] [answered]
It doesn't look like this picks up security patches.  Is that correct or am I doing something wrong
Update-DbaInstance -ComputerName xxxxxx -Version 2016 -Path "c:\temp\2016" -Restart -Confirm:$false

**Comment by wsmelton** [comment]
If your are referring to the latest security bulletin you will need to run the command to update the index. Those were added last week.

The index shipped with the module is only updated when we do another release.

Ref: https://github.com/sqlcollaborative/dbatools/wiki/Asset-Update-Process

**Reply by KezHalls** [reply]
Hi, sorry should have mentioned I updated the index first 
and if I do this: Get-DbaKbUpdate -name KB4583461
it shows details on the kb
But if I run the command 
Update-DbaInstance -ComputerName xxxxxx -Version 2016 -Path "c:\temp\2016" -Restart -Confirm:$false
I get
WARNING: [11:09:38][Get-SqlInstanceUpdate] 13.0.5850 on computer [xxxx] is already the latest available.

Which it is not
current sql version Microsoft SQL Server 2016 (SP2-CU15) (KB4577775)

**Reply by wsmelton** [reply]
That I'm not sure... @nvarscar any thoughts?

**Reply by wsmelton** [reply]
Actually 13.0.5850 = KB4577775 @KezHalls 

The security patch is under `kb4583461`: https://support.microsoft.com/kb/4583461 So technically latest for SQL Server 2016 is SP2 CU15+GDR but it does not appear our build ref handles that properly. // @niphlod

**Reply by wsmelton** [reply]
We list the KB but that latest flag not being checked I think is what effects the update instance command

![image](https://user-images.githubusercontent.com/11204251/104976855-cd223d00-59c3-11eb-8f8c-aed7ee287fe9.png)

**Comment by nvarscar** [comment, marked as answer]
@wsmelton is correct that the security patch is never considered latest. If you want to install a specific security patch, you can use the `-KB 4577775` parameter that would explicitly install this particular KB.

**Reply by KezHalls** [reply]
thanks.  will change my code to handle this way then.
